python/lsst/ts/IntegrationTests/base_script.py

```diff
@@ -38,10 +38,6 @@ class BaseScript:
 
     Parameters
     ----------
-    config : `str`
```

Comment:
Didn't catch this before, but the Parameters section here belongs to the `__init__` function.